envs/example/defaults.yml

```diff
@@ -21,6 +21,7 @@ secrets:
   horizon_secret_key:           asdf
   glance_sync:      ADQ64XUQLUWH75M634RVBLP55RKPGGOWG
   monitor_password: asdf
+  telemetry_password: asdf
```

Comment:
telemetry_password is meant for rabbit, whereas service_password is for os & keystone authentication. we can use service_password if it is preferred to keep the same password. check ceilometer.conf under [publisher] group